formats/image-dimension-stacking-plugin/README.md

```diff
@@ -0,0 +1,48 @@
+# Image dimension stacking(0.1.0-dev)
+
+This plugin leverages the [filepattern](https://filepattern2.readthedocs.io/en/latest/Home.html) library and employs the `groupBy` variable to enable the matching of image filenames, facilitating their stacking into multi-dimensional images.
+Currently, the plugin supports the following dimensions and user can choose the relevant variable for the `groupBy` input argument.
+1. multi-channel  `groupBy=c`\
+   For example `filePattern=x01_y01_p01_c{c:d+}.ome.tif`
+2. multi-zplanes  `groupBy=z`\
+   For example `filePattern=tubhiswt_C1-z{z:d+}.ome.tif`
+3. multi-timepoints  `groupBy=t`\
+   For example `filePattern=img00001_t{t:d+}_ch0.ome.tif`
```

Comment:
Add examples for when we have multiple variables and dimensions that are being stacked at once.